src/Symfony/Bundle/FrameworkBundle/CHANGELOG.md

```diff
@@ -6,6 +6,7 @@ CHANGELOG
 
  * Add support for setting `headers` with `Symfony\Bundle\FrameworkBundle\Controller\TemplateController`
  * Derivate `kernel.secret` from the decryption secret when its env var is not defined
+ * Add exception.x.log_channel
```

Comment:
this is a bit cryptic, can't we make it more understandable?

Reply:
Done